bin/crew

```diff
@@ -1297,7 +1297,7 @@ def install
   # remove it just before the file copy
   if @pkg.in_upgrade
     puts 'Attempting removal since this is an upgrade or reinstall...'
-    Command.remove(@pkg)
+    Command.remove(@pkg, verbose: CREW_VERBOSE, force: @opt_force)
```

Comment:
Why not use `@opt_verbose` for consistency?

Reply:
As it stands we don't have an `@opt_verbose` variable, only the constant. (something to do with `lib/fixup.rb`, maybe? i can't remember why the constant was added in favour of the variable)

Reply:
Constants hold their value globally but the scope of instance variables are limited.

Reply:
true-- it was probably because we didn't pass anything to lib/fixup.rb, so the only way to make it verbose was through a constant. I'll see what I can do about that when i refactor `lib/fixup.rb`.